tests/features/maxlength.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Check the feature Maxlength
+  In order to check the functionality of limiting and validating their maximum length in the edit form.
+  As an administrator
+  I want to check the maxlength.
+
+  @api
+  Scenario: Contributor User can check the maxlength counts (without the tags and specifics characters) and Administrator User can check his bypass.
```

Comment:
specifics characters => specific characters

Besides of that, the scenario description needs some more clarification IMHO.
Which case is exactly tested with the example with 10 times &?